sklearn/tests/test_base.py

```diff
@@ -436,6 +436,7 @@ def __init__(self, attribute_pickled=5):
 
     def __getstate__(self):
         data = self.__dict__.copy()
+        data["_sklearn_version"] = sklearn.__version__
```

Comment:
@lesteve I reworked my comment and [added it to the original ticket](https://github.com/scikit-learn/scikit-learn/issues/7135#issuecomment-285683380) because this PR is probably not the best place for this discussion.